Venkat/03.10.2018/task2.js

```diff
@@ -0,0 +1,30 @@
+var arr = ['xl', 'l', 'xxl', 'm', 's'];
+var arr1 = [];
+console.log('-----before sorting------')
+console.log(arr)
+function sortBySize(arr){
+    for( i=0; i<arr.length; i++){
+        if(arr[i]=='s'){
```

Comment:
do use so many if conditions it will lead to cyclomatic complexity plz read the following article for more details. ( https://en.wikipedia.org/wiki/Cyclomatic_complexity)